liquibase-couchbase/src/main/java/liquibase/ext/couchbase/validator/MutateInInsertUpsertUniqueValidator.java

```diff
@@ -15,10 +15,9 @@ public MutateInInsertUpsertUniqueValidator(MutateInType mutateInType) {
     }
 
     @Override
-    public void validate(String path, Value value, List<Value> values) {
+    public void validate(String path, List<Value> values) {
         validatePathPresence(path);
-        validateValuePresence(value);
-        validateNoMultipleValues(values);
+        validateMultipleValuesPresence(values);
```

Comment:
Don't we break the logic here? we need to ensure that if we need to insert or upsert via mutate in, we can pass only 1 value, here you delete ` validateNoMultipleValues(values);` it means that if we pass multiple values, there will no be an error. 

We should validate that only 1 value is passed (that list contains only 1 value)


You need to create method like validateOneValue where you will check that list contains only 1 value

Reply:
Corrected, thanks!